doc/overview/what_is_ccf.rst

```diff
@@ -6,7 +6,7 @@ This page gives a broad overview of the fundamentals of CCF. This is where you s
 CCF in a Hundred Words
 ----------------------
 
-The Confidential Consortium Framework (CCF) is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
+The `Confidential Consortium Framework (CCF) <https://ccf.microsoft.com/>`_ is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
```

Comment:
```suggestion
The `Confidential Consortium Framework (CCF) <https://ccf.dev/>`_ is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
```

Reply:
`ccf.dev` automatically redirects to `ccf.microsoft.com` so it feels more natural to refer to the final URL instead?

Reply:
It does now, but that's an implementation detail. The person who did the website also suggested it was better for SEO to use ccf.dev.